### DISCUSSION TITLE
How to create a project for the latest (8.4.3) version of serenity

### DISCUSSION THREAD

**Discussion by esitehoster** [serenity-is/Serenity#7108] [Help]
Hi Team,
is there any way to create a project for the latest Serenity version  8.4.3
Please guide us, thanks

**Comment by VictorTomaili** [comment]
check this https://github.com/orgs/serenity-is/discussions/7101#discussioncomment-8876323 and this https://github.com/serenity-is/Serenity/issues/7020

**Reply by esitehoster** [reply]
is there any simple guide to directly create a project for 8.4.3, the provided guide shows first create a .net 6 project then update it to .net 8, and then dig to update it to 8.4.3, (but the document is missing how to update it to 8.4.3),
 Now more than 6 months have passed since releasing .net 8, but Seren has an old .net 6 version

**Reply by VictorTomaili** [reply]
If we had them, I would have already shared them with you instead of these links. After upgrading to 8.x you can follow these release notes to update to the latest. https://serenity.is/docs/release-notes/8.0.0

Dotnet 6 is still a supported LTS version. It is not old.

![image](https://github.com/serenity-is/Serenity/assets/2023440/0ef811c8-9efa-4e83-bc70-97ec7496bdaa)
